cmake/sysbuild/image_signing.cmake

```diff
@@ -137,6 +137,22 @@ function(zephyr_mcuboot_tasks)
     set(input ${APPLICATION_BINARY_DIR}/zephyr/${KERNEL_NAME})
   endif()
 
+  if(SB_CONFIG_SOC_SERIES_NRF54LX AND SB_CONFIG_BOOT_SIGNATURE_TYPE_ED25519)
```

Comment:
You would need to force usage of imgtool in ncs for this, look above for compression to see where, because the SDK toolchain is imgtool 2.1.0 which does not have this feature

Reply:
OK, so I have been able to avoid adding too many Kconfig options allover the place, by searching the imgtool extras for things that require our specific tool.

Reply:
No, imgtool extra is for user control, required options should not be there, use Kconfigs